Tärinämittaus.md

```diff
@@ -35,36 +34,35 @@ Käyttöliittymän kansiorakenne:
 
 ## Git
 
-Tällä hetkellä kaikki koodi on repositoriossa <https://github.com/finnishtransportagency/Tarinamittaus-Backend> Projektin toinen branchi <https://github.com/finnishtransportagency/Tarinamittaus-UI> on tällä hetkellä vielä tyhjä, otetaan käyttöön siinä vaiheessa kun migraatio on tehty aws:sään.
+Sovelluskoodi on repositoriossa <https://github.com/finnishtransportagency/tarinamittaus>.
+Infrakoodi: https://github.com/finnishtransportagency/tietokatalogi-infra
 
 ### Branchit ja workflow
 
-Työbranchi (pyritään nimeämään ANALPK-XXXX_lyhytkuvaus) -> master -> test2 -> production
+Työbranchi (pyritään nimeämään ANALPK-XXXX_lyhytkuvaus) -> dev -> master
```

Comment:
Pitäskö tässä olla maininta mistä bränchistä lohkaistaan työbränchi?

Reply:
Lisätty maininta. Lisäsin myös ohjeen, että muutokset devistä masteriin tehdään mieluiten ilman merge committia eli fast forward -tyyliin, niin master ei mene vahingossa devin edelle. Voidaan toki miettiä näitä tarkemminkin jos haluat.